### ISSUE TITLE
[2019-12-7] The Self-written BatchNorm Perform's grad Not Consistent with nn.BatchNorm2d 

### ISSUE THREAD

**Issue by A-suozhang** [A-suozhang/my-debug-record#1]
##  Bug Report

* The Self-Written BN 前向传播可以做到和原生BN一致,但是传播回来的梯度不正确(且会和Batch Size成正相关),会差到多个数量级(1e-4 - 1e-7)
